torch/csrc/jit/codegen/cuda/test/test_gpu2.cpp

```diff
@@ -9040,21 +9040,13 @@ TEST_F(NVFuserTest, FusionChannelsLastParser_CUDA) {
 __global__ void CUDAGeneratedKernel(Tensor<__half, 4> T0, Tensor<__half, 4> T2, Tensor<__half, 4> T7) {
   int64_t i143;
   i143 = (((nvfuser_index_t)blockIdx.x) * 128) + ((nvfuser_index_t)threadIdx.x);
-  int64_t i94;
-  i94 = T0.size[2] * T0.size[3];
   int64_t i95;
-  i95 = T0.size[1] * i94;
-  int64_t i150;
-  i150 = T0.size[2] * T0.size[1];
-  int64_t i145;
-  i145 = i143 % i95;
-  int64_t i147;
-  i147 = i145 % i94;
```

Comment:
Some reuse is lost, should be recoverable by https://github.com/csarofeen/pytorch/pull/2254